web_keyboard_navigation/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "web_keyboard_navigation",
+    "version": "11.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
```

Comment:
@daramousk Add `website` key as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)